test/regression.py

```diff
@@ -3353,12 +3366,33 @@ def testatommasses(self):
     "Psi4/Psi4-beta5/water_ccsd.out":   GenericCCTest,
     "Psi4/Psi4-beta5/water_mp2.out":    GenericMP2Test,
 
+    "QChem/QChem4.2/C_bigbasis.out":         QChemBigBasisTest,
+    "QChem/QChem4.2/MoOCl4_sp.out":          GenericCoreTest,
+    "QChem/QChem4.2/Trp_polar.out":          ReferencePolarTest,
+    "QChem/QChem4.2/dvb_bomd.out":           GenericBOMDTest,
+    "QChem/QChem4.2/dvb_gopt.out":           GenericGeoOptTest,
+    "QChem/QChem4.2/dvb_ir.out":             QChemIRTest,
+    "QChem/QChem4.2/dvb_raman.out":          QChemRamanTest,
+    # also GenericBasisTest
+    "QChem/QChem4.2/dvb_sp.out":             GenericSPTest,
+    "QChem/QChem4.2/dvb_sp_un.out":          GenericSPunTest,
+    "QChem/QChem4.2/dvb_td.out":             QChemTDDFTTest,
+    "QChem/QChem4.2/water_ccd.out":          GenericCCTest,
+    "QChem/QChem4.2/water_ccsd(t).out":      GenericCCTest,
+    "QChem/QChem4.2/water_ccsd.out":         GenericCCTest,
+    "QChem/QChem4.2/water_cis.out":          QChemCISTest,
+    # ???
+    # create mode 100644 QChem/QChem4.2/water_ir.out
+    # create mode 100644 QChem/QChem4.2/water_ir_anharm.out

```

Comment:
Here too.

Reply:
These were files copied over that didn't correspond to any tests in `testdata`. I think they may be from when we were thinking about parsing anharmonic frequencies. I've just left them in in the regressions repo.